src/server/llmQueries.ts

```diff
@@ -117,8 +128,10 @@ const META_DESCRIPTION_QUERY: MetaDescriptionQuery = {
         <Instructions>
           You have an extensive experience from the educational sector.
           You are specialized in finding good ways to describe educational articles, in a way that the reader can understand what the article is about from a very short summary.
-          Your task is to read through the article within the <draft> tag with title ${title} and write meta description in EN.
-          The meta description must be in less than 15 words and written inside a <answer>-tag.
+          You have been tasked to read through the article withing the <draft> tag and write a meta description in EN.
```

Comment:
`Your task is to read through the article within the <draft> tag, and write a meta description about it`